package.json

```diff
@@ -30,11 +32,50 @@
   },
   "homepage": "https://github.com/logdna/logdna-winston#readme",
   "dependencies": {
-    "logdna": "^3.5.0",
-    "winston": "^3.2.1"
+    "@logdna/logger": "^1.3.2"
   },
   "devDependencies": {
-    "babel-eslint": "^10.0.3",
-    "eslint": "^6.8.0"
+    "eslint": "^7.10.0",
+    "eslint-config-logdna": "^2.0.0",
+    "nock": "^13.0.4",
+    "tap": "^14.10.8",
+    "winston": "^3.3.3",
```

Comment:
@darinspivey, I remember someone mentioned somewhere using this as one of `peerDependencies` - why don't we do this now?

Reply:
I think `peerDependencies` are magical and confusing.  Since this is only used for the test suite, I think it's better as an explicit dev dep.

Reply:
well, there's not much magic to it IMO - it just _signals_ that "this other dependency is expected to exist, and this package only works with version N of it". I think starting in npm 3 they stopped automatically installing peer deps.